docs/topics/db/transactions.txt

```diff
@@ -210,6 +210,14 @@ Django provides a single API to control database transactions.
             if obj.active:
                 ...
 
+        This also applies to any other mechanism that may hold app state, such
+        as caching or global variables. For example, if the code proactively
+        updates data in the cache after saving an object, it's recommended to
+        use :ref:`transcation.on_commit <performing-actions-after-commit>`
```

Comment:
`transcation.on_commit()` parentheses are not automatically added when using `:ref:`.